norfair/video.py

```diff
@@ -134,6 +172,19 @@ def _fail(self, msg: str):
         exit()
 
     def write(self, frame: np.array) -> int:
+        """
+        Function to which the frames which will compose the output video should get passed to.
```

Comment:
In general, let's try to drop the "Function which" or "Function to", which only add verbosity.

From this docstring, it's still not clear what the function does (you have to infer it from the name), so the docstring needs a rewrite.